zerver/openapi/curl_param_value_generators.py

```diff
@@ -252,6 +252,19 @@ def create_user() -> dict[str, object]:
     }
 
 
+@openapi_param_value_generator(["/users/{email]:patch", "/users/{user_id}:patch"])
+def new_email_value() -> dict[str, object]:
+    count = 0
+    exists = True
+    while exists:
+        email = f"new{count}@zulip.com"
```

Comment:
Always use `@example.com` for fake email addresses; it's an RFC-specific standard for this.

Reply:
I guess changing this is not actually helpful given the Shakespeare organization, leaving as-is.